hawc/apps/animalv2/models.py

```diff
@@ -0,0 +1,576 @@
+    dataset_type = models.CharField(
+        blank=True, default="", max_length=2, choices=constants.DatasetType.choices
+    )
+    variance = models.FloatField(blank=True, null=True, validators=[MinValueValidator(0)])
+    statistical_power = models.CharField(max_length=128, blank=True, help_text="TODO")
+    statistically_significant = models.PositiveSmallIntegerField(
+        choices=constants.StatisticallySignificant.choices
+    )
+    NOAEL = models.SmallIntegerField(default=-999, help_text="No observed effect level")
+    LOAEL = models.SmallIntegerField(default=-999, help_text="Lowest observed effect level")
+    response_units = models.CharField(
+        max_length=32,
+        blank=True,
+        help_text="Units the response was measured in (i.e., \u03BCg/dL, % control, etc.)",
+    )
+    dose_response_observations = models.TextField(help_text="TODO")
+    result_details = models.TextField(help_text="TODO")
+
+    created = models.DateTimeField(auto_now_add=True)
+    last_updated = models.DateTimeField(auto_now=True)
+
+    def get_assessment(self):
+        return self.experiment.get_assessment()
+
+    def get_study(self):
+        return self.experiment.get_study()
+
+    # TODO - flat_complete_header_row / flat_complete_data_row
+
+
+"""
+based on conversations with Vidhi and Pam - various fields here (n, treatment_name, loael, etc.)
+are duplicated on both the parent dataExtraction object and these associated group/animal level data
+objects. Not sure how this is supposed to work - does the dataExtraction level field take precedence?
+Provide a default that can be overridden? Still chatting with them to see what's expected here so
+that type of overlap is the most likely portion of the model to be subject to change.
+"""
+
+
+class DoseResponseGroupLevelData(models.Model):
+    objects = managers.DoseResponseGroupLevelDataManager()
+
+    data_extraction = models.ForeignKey(
+        DataExtraction, on_delete=models.CASCADE, related_name="v2_group_level_data"
+    )
+    treatment = models.ForeignKey(
+        Treatment, on_delete=models.CASCADE, related_name="v2_group_level_data"
+    )
+    # specifically, DoseGroup.dose?
+    dose = models.ForeignKey(
+        DoseGroup, on_delete=models.CASCADE, related_name="v2_group_level_data"
+    )
+    n = models.PositiveSmallIntegerField(blank=True, null=True, validators=[MinValueValidator(0)])
+    response = models.FloatField()
+    variance = models.FloatField(blank=True, null=True, validators=[MinValueValidator(0)])
+    treatment_related_effect = models.PositiveSmallIntegerField(
+        choices=constants.TreatmentRelatedEffect.choices
+    )
+    statistically_significant = models.PositiveSmallIntegerField(
+        choices=constants.StatisticallySignificant.choices
+    )
+    p_value = models.CharField(max_length=128, blank=True, help_text="TODO")
+    NOAEL = models.SmallIntegerField(default=-999, help_text="No observed effect level")
```

Comment:
remove noel/loel from this table - rename in all cases to noel/loel as the "a" is assessment specific.

Reply:
There's an assessment-level setting that changes how those values are named, https://github.com/shapiromatron/hawc/blob/213b0a2121bcbbc34ec098bdddcc791aaae717f9/hawc/apps/assessment/models.py#L260-L265.  

I haven't heard of a case where the same assessment needs both loels and loaels, etc.